api/src/main/java/io/druid/data/input/impl/StringInputRowParser.java

```diff
@@ -37,6 +38,7 @@
 
 /**
  */
+@PublicApi
 public class StringInputRowParser implements ByteBufferInputRowParser
```

Comment:
Maybe nobody really uses it though and we can take out the annotation. Hmm, ok I'll take it out.